R/slice.R

```diff
@@ -153,6 +153,13 @@ slice_tail.data.frame <- function(.data, ..., n, prop) {
   slice(.data, idx(dplyr::n()))
 }
 
+check_slice_by <- function(idx, order_by, n) {
+  if (!is.null(order_by)) {
+    vec_assert(order_by, size = n)
```

Comment:
```suggestion
    vec_assert(order_by, size = n, arg = "order_by")
```

Since I think users could see this?

Reply:
Although I guess `vec_assert()` will do that for you...